src/components/BookingSeat.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from "react";
```

Comment:
Refactor this component to smaller components and separate logic from presentation.